torchci/lib/bot/drciBot.ts

```diff
@@ -30,8 +30,8 @@ async function getDrciComment(
 }
 
 export function formDrciComment(prNum: number): string {
-  let body = "# :link: Helpful Links\n";
-  body += `* :test_tube: See artifacts and rendered test results [here](${hudUrl}${prNum})\n`;
+  let body = "## :link: Helpful Links\n";
+  body += `### :test_tube: See artifacts and rendered test results [here](${hudUrl}${prNum})\n`;
```

Comment:
I don't think you have to do this += thing with the \n's The ` quotes should handle multi line things and add the enters.